dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/gdv/model/soziales/einkommensgleichheit/UeberwachungDerEinkommensungleichheitValues.kt

```diff
@@ -0,0 +1,13 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.gdv.model.soziales.einkommensgleichheit
+
+import java.math.BigDecimal
+
+/**
+ * Data class for the timeseries data contained in the field ueberwachungDerEinkommensungleichheit
+ */
+data class UeberwachungDerEinkommensungleichheitValues(
```

Comment:
Name?

Reply:
ist autogeneriert

Reply:
Yes, but why is this suffix necessary?

Reply:
I personally like the suffix, because otherwise you might think this data class is actually directly used for a field, when in fact it is used to display the correct rows in a rolling window.

Reply:
ok, i yield :)